apps/web/test/lib/getSchedule.test.ts

```diff
@@ -16,157 +7,28 @@ import {
   createOrganization,
   getOrganizer,
   getScenarioData,
+  Timezones,
+  TestData,
 } from "../utils/bookingScenario/bookingScenario";
 
+import { describe, vi, test } from "vitest";
+
+import dayjs from "@calcom/dayjs";
+import type { BookingStatus } from "@calcom/prisma/enums";
+import { getAvailableSlots as getSchedule } from "@calcom/trpc/server/routers/viewer/slots/util";
+
+import { expect } from "./getSchedule/expects";
+import { setupAndTeardown } from "./getSchedule/setupAndTeardown";
+import { timeTravelToTheBeginningOfToday } from "./getSchedule/utils";
+
 vi.mock("@calcom/lib/constants", () => ({
   IS_PRODUCTION: true,
   WEBAPP_URL: "http://localhost:3000",
   RESERVED_SUBDOMAINS: ["auth", "docs"],
 }));
 
-declare global {
-  // eslint-disable-next-line @typescript-eslint/no-namespace
-  namespace jest {
-    interface Matchers<R> {
-      toHaveTimeSlots(expectedSlots: string[], date: { dateString: string }): R;
-    }
-  }
-}
-
-expect.extend({
-  toHaveTimeSlots(
-    schedule: { slots: Record<string, Slot[]> },
-    expectedSlots: string[],
-    { dateString }: { dateString: string }
-  ) {
-    if (!schedule.slots[`${dateString}`]) {
-      return {
-        pass: false,
-        message: () => `has no timeslots for ${dateString}`,
-      };
-    }
-    if (
-      !schedule.slots[`${dateString}`]
-        .map((slot) => slot.time)
-        .every((actualSlotTime, index) => {
-          return `${dateString}T${expectedSlots[index]}` === actualSlotTime;
-        })
```

Comment:
Refactored out certain things to be shareable across multiple files. 
Makes it easy to test getSchedule keeping the code in different files.